src/main/java/com/squareup/kotlinpoet/TypeName.kt

```diff
@@ -412,7 +412,7 @@ class LambdaTypeName private constructor(
   }
 
   fun copy(
-    nullable: Boolean = this.nullable,
+    nullable: Boolean = this.isNullable,
```

Comment:
Should the `copy` parameter also be renamed to `isNullable` for consistency?

Reply:
I think `isNullable` reads better when it's an accessor ("is this TypeName nullable?"), and `nullable` reads better when it's a parameter ("I'm creating a nullable TypeName").

Reply:
Hmm but then the constructor parameter is `isNullable` 🙂

Reply:
`TypeName` is abstract and its subtypes don't have public constructors

Reply:
Oops missed that!